www/views/compare.html

```diff
@@ -0,0 +1,200 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>HeatpumpMonitor.org</title>
+<script src="https://code.jquery.com/jquery-3.6.3.min.js"></script>
+  <link rel="stylesheet" href="views/style.css?v=8" />
+  <link href="https://openenergymonitor.org/homepage/theme/favicon.ico" rel="shortcut icon">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/fontawesome.min.css">
+	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/solid.min.css">
+  <script src="https://cdn.plot.ly/plotly-2.18.1.min.js" charset="utf-8"></script>
+</head>
+
+<body>
+
+<div id="header">
+  <div id="title"><span class="big">
+    <b>HeatpumpMonitor</b>.org</span></div>
+  <div id="navigation">
+    <a href="." title="Home"><i aria-hidden="true" class="fas fa-home"></i></a>
+    <a href="stats.html" title="30 Day Stats"><i aria-hidden="true" class="fas fa-table"></i></a>
+    <a href="costs.html" title="Running Costs"><i aria-hidden="true" class="fas fa-coins"></i></a>
+    <a href="graph.html" title="Comparison Charts" class="active"><i aria-hidden="true" class="fas fa-chart-line"></i></a>
```

Comment:
We're going to have to split this into two links, and think of better titles, or we have all the chart features on one page.

The hardest part is picking an appropriate icon. I'm picking from [fontawesome v5.15](https://fontawesome.com/v5/search?o=r&s=solid), and best I can find is `<i class="fas fa-object-group"></i>`. v6 has fa-chart-scatter, but I couldn't get it to work.

Reply:
ok sounds good, I think getting all the chart features on the same page may be too much of a challenge for now. 
I think there will be another graph page shortly to provide the half hourly profile for all systems...